wiki/Ranking_Criteria/en.md

```diff
@@ -118,7 +118,7 @@ Important to understand before using:
   - Note: For tracks which are both cut and sped up, combine their markers into `(Sped Up & Cut Ver.)` or `(Nightcore & Cut Ver.)`.
 - **Special unicode characters must be filtered to their nearest standard equivalent or removed from the romanised fields within a `.osu` file.** `★ ☆ ⚝ ✪` and the likes are substituted to an asterisk (`*`). Other special characters are to be romanised or dropped on case-by-case basis.
 - **If a beatmap's track is composed of two or more songs, list the song titles clearly with a dividing symbol inbetween or use a title descriptive of its contents.** If the title becomes too long as a result, a descriptive title must be used instead.
-- **If a symbol is used to group parts of a title, a space must be used before and after the group, but not directly before or after the symbols within the groups.**
+- **Symbols used to group a part of the title must include surrounding spaces.** For example, `Song★Title★` becomes `Song ★Title★`.
```

Comment:
```suggestion
- **Symbols used to group part of a title must include surrounding spaces.** For example, `Song★Title★` becomes `Song ★Title★`.
```

wala's originally wording for this part seemed fine, not sure why "the" is used when it's not referring to anything in particular